config/api/models/Page.php

```diff
@@ -74,7 +81,14 @@
         'parents' => function (Page $page) {
             return $page->parents()->flip();
         },
+        /**
+         * @deprecated 3.6.0
+         * @todo Throw deprecated warning in 3.7.0
+         * @todo Remove in 3.8.0
+         * @codeCoverageIgnore
+         */
         'prev' => function (Page $page) {
+            deprecated('The `prev` field has been deprecated and will be removed in 3.7.0');
```

Comment:
@distantnative You removed `deprecated()` method from `next` field. This line should be removed?

Reply:
Oh yes, good catch! I think we are in no rush to remove them. So to make it easier for users, I think it's better to only mark them as deprecated now, then start throwing warning in 3.7 and finally remove it in 3.8